src/organizer.js

```diff
@@ -63,7 +63,12 @@ class Organizer {
           console.log(`send successful for ${JSON.stringify(userA)} and ${JSON.stringify(userB)}`);
           return this.recordMeeting(userA, userB);
         })
-        .catch(() => {
+        .catch((err) => {
+          if (err && err.response && err.response.data) {
```

Comment:
We're using `error` in other places in the project. Can you use that instead of `err` for consistency?

Reply:
Yes, I have already changed it and Now trying to format code using eslint